grype/vulnerability_matcher.go

```diff
@@ -93,7 +94,10 @@ func (m *VulnerabilityMatcher) findDBMatches(pkgs []pkg.Package, context pkg.Con
 	var ignoredMatches []match.IgnoredMatch
 
 	log.Trace("finding matches against DB")
-	matches := m.searchDBForMatches(context.Distro, pkgs, progressMonitor)
+	matches, err := m.searchDBForMatches(context.Distro, pkgs, progressMonitor)
```

Comment:
this is where the error is surfaced from my previous comment for the new `indexFalsePositivesByLocation` error